dokka-integration-tests/cli/build.gradle.kts

```
@@ -15,14 +15,18 @@ dependencies {
     implementation(projects.utilities)
 }
 
-/* Create a fat base plugin jar for cli tests */
-val basePluginShadow: Configuration by configurations.creating {
+// Configuration for plugins/dependencies required to run CLI with base plugin
+val cliPluginsClasspath: Configuration by configurations.creating {
     attributes {
         attribute(Usage.USAGE_ATTRIBUTE, project.objects.named(Usage::class.java, "java-runtime"))
     }
+
+    // we don't fetch transitive dependencies here to be able to control external dependencies explicitly
+    isTransitive = false
 }
 
-val cliConfiguration: Configuration by configurations.creating {
+// Configuration for CLI jar
+val cliClasspath: Configuration by configurations.creating {
```

```suggestion
val cliClasspath: Configuration by configurations.creating {
    description = "dependencies for CLI JAR"
```

Suggestion: use description to document the usage, so it's visible in more places.

---

Thanks!